README.md

```diff
@@ -94,97 +95,86 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+> Key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Add new categories and delete it**
+- **Add new transactions and delete it**
+- **Display details of transactions**
+- **You can see how much money you spent**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- [Live Demo Link](https://hana-kumie-1212.herokuapp.com/)
```

Comment:
- [ ] Kindly see this article on [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/10903290796179) on how to deploy your application on render, It would be better to deploy your application so your app is available online, please use Netlify, render or any other free service